typescript/codepipeline-using-cross-account-repos/README.md

```diff
@@ -0,0 +1,41 @@
+# My Widget Service
+<!--BEGIN STABILITY BANNER-->
+---
+
+![Stability: Stable](https://img.shields.io/badge/stability-Stable-success.svg?style=for-the-badge)
+
+> **This is a stable example. It should successfully build out of the box**
+>
+> This examples does is built only on the CDK core library, and does not have any infrastructure prerequisites to build.
+
+---
+<!--END STABILITY BANNER-->
+
+This example shows show you can make a pipeline with cross account repos
```

Comment:
This should be more explanatory. Which account is dev? Which is tools? What is in each account? A diagram is generally useful and I think one already exists in the ref-arch cfn repo.